NOTICE

```diff
@@ -5,3 +5,34 @@ This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
 The initial codebase was donated to the ASF by HugeGraph Authors, copyright 2017-2021.
+
+
+// ------------------------------------------------------------------
+// NOTICE file corresponding to the section 4d of The Apache License,
+// Version 2.0, in this case for JanusGraph
+// ------------------------------------------------------------------
+
+JanusGraph
+Copyright (C) 2002, 2016, Oracle  and/or its affiliates. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License") reproduced below or available at:
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+
+// ------------------------------------------------------------------
+// NOTICE file corresponding to the section 4d of The Apache License,
+// Version 2.0, in this case for Apache TinkerPop
+// ------------------------------------------------------------------
+
+Apache TinkerPop
```

Comment:
according to the order of https://github.com/apache/incubator-hugegraph/pull/2100/files#diff-f8a05717594bf148241760c18447442978fdc9db03a4aae93739529e0c124f8aR92, can we keep TinkerPop first and move to line 8?